doc/rados/configuration/pool-pg-config-ref.rst

```diff
@@ -60,15 +60,6 @@ Ceph configuration file.
 :Default: ``30``
 
 
-``mon pg warn max per osd``
```

Comment:
could you update `qa/tasks/ceph.conf.template` accordingly?

Reply:
OK  thank you